operator/pkg/controller/karmada/controller.go

```diff
@@ -89,6 +99,11 @@ func (ctrl *Controller) ensureKarmada(ctx context.Context, karmada *operatorv1al
 	// then lets add the finalizer and update the object. This is equivalent
 	// registering our finalizer.
 	updated := controllerutil.AddFinalizer(karmada, ControllerFinalizerName)
+	if _, isExist := karmada.Labels[DisableCascadingDeletionLabel]; !isExist {
```

Comment:
I think it does not need to fill in the default label if users not set

Reply:
if there is not a label in the cr,  should we delete karmada instance or not?

Reply:
After I think about it, I think it is acceptable to add a default label.